config.go

```diff
@@ -681,10 +682,24 @@ func (wc WriteConcern) Resolve() *writeconcern.WriteConcern {
 	return writeconcern.New().WithOptions(opts...)
 }
 
+type ReadConcern struct {
+	Level string `yaml:"level"`
+}
+
+func (wc ReadConcern) Resolve() *readconcern.ReadConcern {
+
+	if wc.Level == "majority" {
+		return readconcern.Majority()
+	} else {
+		return readconcern.New()
```

Comment:
Is this a valid readconcern to return? I worry about someone changing code elsewhere and then ending up setting `readconcern.New()`.

Reply:
Not really sure. I was setting it to readconcern.Available(), but this caused a variety of errors of the form "The readConcern level must be either 'local' or 'majority' in order to upconvert the readConcern level to 'snapshot'". This got rid of those errors.

Reply:
I'd suggest either setting a reasonable default, if there is one, or defaulting to majority.

Reply:
@bsamek changed it to default to Majority